index.html

```diff
@@ -26,5 +26,139 @@ <h3 class="profession">I’m a Software Developer</h3>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project1-button">See Project</button>
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
```

Comment:
- [ ] everything looks great. Please kindly make your work responsive (at least 480px ) . when you have a desktop version it will be easier to link to it. nice job
![Screenshot (753)_LI](https://user-images.githubusercontent.com/65298048/212433009-1a8b404b-6d8f-486a-adca-02632c5d41d8.jpg)